boards/iotlab-m3/include/board.h

```diff
@@ -82,6 +82,14 @@ extern "C" {
 #define LSM303DLHC_DRDY     GPIO_PIN(PORT_B,2)
 /** @} */
 
+/**
+ * @brief   ADC configuration for random seed generation
+ * @{
+ */
+#define RANDOM_SEED_ADC_LINE         (0)
+#define RANDOM_SEED_ADC_RES          ADC_RES_12BIT
```

Comment:
I did it on purpose to demonstrate how/where to place these parameters, if needed. However, I moved  this config to stm32f0discovery